docs/reference/filebeat/filebeat-input-entity-analytics.md

```diff
@@ -132,14 +131,14 @@ A user with appropriate permissions must be set up in the Active Directory Serve
 
 #### Overview [_overview]
 
-The Active Directory provider periodically queries the Active Directory server, retrieving updates for users and groups, updates its internal cache of user and group metadata and group membership information, and ships updated user metadata to Elasticsearch.
+The Active Directory provider periodically queries the Active Directory server, retrieving updates for users, devices, and groups, updates its internal cache of user, device, and group metadata and group membership information, and ships updated user and device metadata to Elasticsearch.
```

Comment:
The introduction of more commas might make this more difficult to parse. You could consider breaking this into an unordered list. Something like this:

```suggestion
The Active Directory provider periodically:

* Queries the Active Directory server, retrieving updates for users, devices, and groups.
* Updates its internal cache of user, device, and group metadata and group membership information.
* Ships updated user and device metadata to Elasticsearch.
```